Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -98,14 +133,22 @@ private set
                     else
                     {
                         // Abort press.
-                        isPressing = false;
+                        IsPressing = false;
 
                         TouchEnd.Invoke();
                     }
                 }
             }
         }
 
+        /// <summary>
+        /// Represents the state of whether the button is currently being pressed.
```

Comment:
It would be good to define pressing here. Whether pressing is we are touching the button, or pressing means the button has passed some press distance.

Reply:
that is currently determined in UpdatePressedState which is also invoking the ButtonPressed and Released signals. do we want to move all that into the getter?

Reply:
I'm not a big fan of getter with side effects